libgit/autogit_manager.go

```diff
@@ -438,6 +453,79 @@ func (am *AutogitManager) queueReset(ctx context.Context, req resetReq) (
 	return req.doneCh, nil
 }
 
+func (am *AutogitManager) doDelete(req deleteReq) (err error) {
+	ctx, cancel := context.WithCancel(context.Background())
+	defer cancel()
+	ctx = libkbfs.CtxWithRandomIDReplayable(
+		ctx, ctxIDKey, ctxOpID, am.log)
+
+	am.log.CDebugf(ctx, "Processing delete request of %s/%s/%s",
+		req.dstTLF.GetCanonicalPath(), req.dstDir, req.repo)
+	defer func() {
+		am.deferLog.CDebugf(ctx, "Delete request completed: %+v", err)
+	}()
+
+	// Make a new single-op config for processing this request.
+	ctx, gitConfig, tempDir, err := am.getNewConfig(ctx)
+	if err != nil {
+		return err
+	}
+	// Autogit data should be regular data blocks.
+	gitConfig.SetDefaultBlockType(keybase1.BlockType_DATA)
+	defer func() {
+		gitConfig.Shutdown(ctx)
+		rmErr := os.RemoveAll(tempDir)
+		if rmErr != nil {
+			am.log.CWarningf(
+				ctx, "Error cleaning storage dir %s: %+v\n", tempDir, rmErr)
+		}
+	}()
+
+	uniqID, err := makeUniqueID(ctx, gitConfig)
+	if err != nil {
+		return err
+	}
+
+	dstFS, err := libfs.NewFS(
+		ctx, gitConfig, req.dstTLF, req.dstDir, uniqID,
+		keybase1.MDPriorityNormal)
+	if err != nil {
+		return err
+	}
+
+	canWork, err := am.canWorkOnRepo(ctx, dstFS, req.repo)
+	if err != nil {
+		return err
+	}
+	if !canWork {
+		am.log.CDebugf(ctx,
+			"Another worker is currently in charge; skipping reset")
+		// TODO: retry in a little while?
+		return nil
+	}
+	defer func() {
+		workDoneErr := am.workDoneOnRepo(ctx, dstFS, req.repo, err)
+		if err == nil {
+			err = workDoneErr
+		}
+	}()
+
+	fi, err := dstFS.Stat(req.repo)
+	if err != nil {
+		return err
+	}
+	return recursiveDelete(ctx, dstFS, fi)
+}
+
+func (am *AutogitManager) deleteLoop() {
+	for reqInt := range am.deleteQueue.Out() {
+		req := reqInt.(deleteReq)
+		_ = am.doDelete(req)
```

Comment:
Is it intended here to ignore error without logging?

Reply:
It's logged in a `defer` in `doDelete`.